Source/Core/DolphinLib.vcxproj

```diff
@@ -27,6 +27,12 @@
       <Project>{41279555-f94f-4ebc-99de-af863c10c5c4}</Project>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="InputCommon\ControllerEmu\ControlGroup\SensorBar.cpp" />
```

Comment:
These includes should actually go in https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/DolphinLib.props